app/facades/user_facade.rb

```diff
@@ -0,0 +1,11 @@
+class UserFacade
+  def self.user_info(user_id)
+    json = UserService.user_info(user_id)
+    friends = json[:data][:relationships][:friends].map do |data|
+      Friend.new(data[1])
+    end
+    user = User.new(json[:data])
+    user.add_friends(friends)
+    user
```

Comment:
I have some questions about this, I thought that other users would not be added to the game night until accepting an invite, and if so I think that maybe the Friend poro is unnecessary?

Reply:
I think this was done to invite the friend to the game night. but I could be wrong.